tests/integration/security/authorization_test.go

```diff
@@ -990,6 +1000,7 @@ func TestAuthorization_GRPC(t *testing.T) {
 // with path "/a/../b" should be normalized to "/b" before using in authorization.
 func TestAuthorization_Path(t *testing.T) {
 	framework.NewTest(t).
+		Features("security.authorization.path-get").
```

Comment:
it's not very useful to just call this as `path`, we have tests everywhere covering the path field., Look at the comment and the actual test logic, it's testing `path-normalization`

Reply:
change it to path-normalization

Reply:
change from security.authorization.path-get to security.authorization.path-normalization